data/devices/logitech-g304.device

```diff
@@ -0,0 +1,9 @@
+[Device]
+Name=Logitech Gaming Mouse G304
+DeviceMatch=usb:046d:c53f
+Driver=hidpp20
+DeviceType=mouse
+
+[Driver/hidpp20]
+Leds=0
+Quirk=G304
```

Comment:
'G304' is not a valid quirk, it should could either be 'G305' or not set at all. Our tests don't catch that right now, which I opened an issue for: #1557.

Reply:
Yeah, I'll need to change it to G305 then. It's my first time trying to do this so pardon my skill issue